Library/Formula/dmd.rb

```diff
@@ -65,6 +57,27 @@ def install
     end
   end
 
+  # Fix up dmd.conf to point to the correct include paths.
+  def fix_dmd_conf
+    # Homebrew's etc.install method doesn't let you overwrite etc files,
+    # so edit it in place if it's there:
+    conf = etc/"dmd.conf"
+    if conf.exist?
+      inreplace conf, /^DFLAGS=.+$/, "DFLAGS=-I#{include}/d2 -L-L#{lib}"
+    else
+      conf.write <<-EOS.undent
+        [Environment]
+        DFLAGS=-I#{include}/d2 -L-L#{lib}
+        EOS
+    end
+
+  end
+
+  def post_install
+    # We need to fix up dmd.conf even if users install/upgrade via bottle:
+    fix_dmd_conf
```

Comment:
Ah, I see. Is the `dmd.conf` something users are likely to customise? We could just rename the file to `dmd.conf.old`, notify users if we've done that if it existed and not worry about trying to fix it up at all and just write the correct format.